### PR TITLE
Use Named Keys for API keys

### PR DESCRIPTION
**Pull request by hunterowens**
This PR supplants #144 basedon the new agencies.yml file, but uses named keys rather logic so it is easier to template.

### CODE REVIEW

File: .github/workflows/push_to_gcloud.yml [hunk 1]
```diff
@@ -17,6 +17,15 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.8
+    - name: Install python packages
```
Comment (hunterowens) [inline .github/workflows/push_to_gcloud.yml:20]: this is so cool

**Comment by machow** [comment]
@themightychris it looks like the markdown linter has a false positive for a dead link--have you seen it run into this much?

opening_a_page

**Comment by machow** [comment]
@hunterowens if this works for you, I can loop around and merge tomorrow (may quickly set the action to fill in API keys in `agencies.yml` on every branch, so people get feedback on missing keys when updating)

**Comment by hunterowens** [comment]
sg

**Pull request merged**